lib/tasks/journeys_report.rake

```diff
@@ -67,7 +67,7 @@ namespace :journeys do
           date: move.date,
           from: move.from_location.nomis_agency_id,
           to: move.to_location&.nomis_agency_id,
-          person_id: move.profile.person_id,
+          person_id: move.profile&.person_id,
```

Comment:
you'll need to check for move.profile.person throughout this file (e.g. gender, date_of_birth etc) and update accordingly

Reply:
Included all the references to `profile` in `moves`